kafka-ui-api/src/main/java/com/provectus/kafka/ui/config/auth/LdapSecurityConfig.java

```diff
@@ -97,11 +99,24 @@ public BaseLdapPathContextSource contextSource() {
 
   @Bean
   @Primary
-  public LdapAuthoritiesPopulator ldapAuthoritiesPopulator(BaseLdapPathContextSource contextSource) {
-    var authoritiesPopulator = new DefaultLdapAuthoritiesPopulator(contextSource, props.getGroupFilterSearchBase());
-    authoritiesPopulator.setRolePrefix("");
-    authoritiesPopulator.setConvertToUpperCase(false);
-    return authoritiesPopulator;
+  public DefaultLdapAuthoritiesPopulator ldapAuthoritiesExtractor(ApplicationContext context,
+                                                                  BaseLdapPathContextSource contextSource,
+                                                                  @Nullable AccessControlService acs) {
```

Comment:
how can AccessControlService be null here? We rely on it to be present in all controllers.

Reply:
Not sure actually, but you're absolutely right. That used to by copy-pasted around, I guess might be related to some tests.